### PR TITLE
Coafile

### PR DESCRIPTION
**Pull request by abhsag24**

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

File: MANIFEST.in [hunk 1]
```diff
@@ -1 +1 @@
-include requirements.txt mac-requirements.txt linux-requirements.txt README.md
\ No newline at end of file
+include requirements.txt mac-requirements.txt linux-requirements.txt README.md
```
Comment (nkprince007) [inline MANIFEST.in]: remove `linux-requirements.txt`

**Comment by nkprince007** [comment]
Amend the last commit, please `MANIFEST` not `MAINFEST` :P

**Comment by nkprince007** [comment]
ack 48b986c

**Comment by nkprince007** [comment]
ack 281ca8b

**Comment by nkprince007** [comment]
ack a3e05ad

**Comment by nkprince007** [comment]
ack 281ca8b
ack 48b986c

**Comment by nkprince007** [comment]
ack 281ca8b

**Pull request merged**